judgments/admin.py

```diff
@@ -1,3 +1,8 @@
-# from django.contrib import admin
+from django.contrib import admin
 
-# Register your models here.
+from judgments.models import CourtDates
+
+
+@admin.register(CourtDates)
+class CourtDatesAdmin(admin.ModelAdmin):
```

Comment:
Currently, since the class isnt doing anything different to thee base model admin, this isn't doing anything that the default admin setup would do anyway? If so, I'd suggest removing it, but maybe I am missing something

Reply:
You need to do the `@admin.register` bit for the model to appear in the admin.